transaction-flow.md

```diff
@@ -409,6 +409,8 @@ The valid `bank_code` values can be found on the [GHIPSS website](https://www.gh
 }
 ```
 
+The `reference` can also be provided optionally for MAD::Cash, but if you want to use this functionality please contact us for more details.
```

Comment:
maybe move this below the following paragraph which describes how to obtain the reference in the normal scenarios